webapi/Models/Response/ServiceOptionsResponse.cs

```diff
@@ -1,34 +1,18 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 
 using System.Collections.Generic;
-using System.Linq;
 using System.Text.Json.Serialization;
 
 namespace CopilotChat.WebApi.Models.Response;
 
-public class ServiceOptionsResponse
-{
-    /// <summary>
-    /// Configured memory store.
-    /// </summary>
-    [JsonPropertyName("memoryStore")]
-    public MemoryStoreOptionResponse MemoryStore { get; set; } = new MemoryStoreOptionResponse();
-}
-
 /// <summary>
-/// Response to memoryStoreType request.
+/// Response given by <seealso cref="ServiceOptionsController"/>
 /// </summary>
-public class MemoryStoreOptionResponse
+public class ServiceOptionsResponse
 {
     /// <summary>
-    /// All the available memory store types.
-    /// </summary>
-    [JsonPropertyName("types")]
-    public IEnumerable<string> Types { get; set; } = Enumerable.Empty<string>();
```

Comment:
Isn't this needed to populate the list of supported options in the front-end?

Reply:
It would be. But why do we even want to know all the other types of possible memory backings? The list keeps expanding and the UI will get more cluttered as we go while we only support using one memory type at a time.  I'm thinking just display what we are using and forget what we are not using.

Reply:
I understand your point, however chat-copilot is useful as a way to demonstrate the capabilities of semantic kernel. One thing we are focusing on is working with other platforms and this is a great way to demonstrate this. I agree that as the list grows, we will need a different way to show this off.

Reply:
Reverting this